bigtable/bttest/inmem_test.go

```diff
@@ -1371,7 +1371,7 @@ func populateTable(ctx context.Context, s *clientIntf, exists bool) error {
 	if !exists {
 		newTbl := btapb.Table{
 			ColumnFamilies: map[string]*btapb.ColumnFamily{
-				"cf0": {GcRule: &btapb.GcRule{Rule: &btapb.GcRule_MaxNumVersions{1}}},
+				"cf0": {GcRule: &btapb.GcRule{Rule: &btapb.GcRule_MaxNumVersions{MaxNumVersions: 1}}},
```

Comment:
mainly fixing problems from checkgofmt target

Reply:
We definitely do not want to do this.  This will just increase our level of fork from google's bttest.  Instead we should disable this particular check for test code.

Reply:
got it, I forgot it was a fork, Im gonna revert this and update the target for bigtable